### PR TITLE
Adding documentation for seedimage

### PR DESCRIPTION
**Pull request by frelon**
Replace the manual ISO creation in quickstart with seed-image resource.

Add page for seedimage reference.

### CODE REVIEW

**Comment by frelon** [comment]
@nunix are you also doing changes to the quickstart? just wanted to give a heads up :+1:

**Review by fgiudici — COMMENTED** [review]
Looks good here!
The changes should end up on the "cli" quickstart only of @nunix quickstart rework .
I would anyway wait for @nunix review before merging.

File: docs/quickstart-cli.md
Comment (davidcassany) [inline docs/quickstart-cli.md:79]: This URL seams to be broken within the preview

**Review by davidcassany — APPROVED** [review]
LGTM, just small comment on a URL

File: docs/quickstart-cli.md
Comment (frelon) [reply docs/quickstart-cli.md]: Good catch!

**Pull request merged**
